pkg/actors/internal/placement.go

```diff
@@ -181,11 +181,12 @@ func (p *ActorPlacement) Start() {
 			p.clientLock.RLock()
 			clientStream := p.clientStream
 			p.clientLock.RUnlock()
-			resp, err := clientStream.Recv()
+
 			if p.shutdown.Load() {
 				break
 			}
 
+			resp, err := clientStream.Recv()
```

Comment:
why was this moved after the `break`?

Reply:
before the current change it was necessary to avoid race conditions between Stop() and this, but now there are no reasons to keep this, so I'm going to put it back there, thanks